reference/7.4/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -16,42 +16,34 @@ the scope of elements.
 
 PowerShell protects access to variables, aliases, functions, and PowerShell
 drives (PSDrives) by limiting where they can be read and changed. PowerShell
-uses scope rules to ensure that you don't inadvertently change an item that
+uses scope rules to ensure that you don't accidentally change an item that
 shouldn't be changed.
 
+## Scope rules
+
+When you start PowerShell, the host (`pwsh.exe`) creates a PowerShell runspace.
+Host processes can have multiple runspaces. Each runspace has its own session
+state and scope containers. Session state and scopes can't be accessed across
+runspace instances.
+
 The following are the basic rules of scope:
 
 - Scopes may nest. An outer scope is referred to as a parent scope. Any nested
   scopes are child scopes of that parent.
-
-- An item is visible in the scope that it was created in and in any child
-  scopes, unless you explicitly make it private.
+- An item is visible in the scope that it was created and in any child scopes,
+  unless you explicitly make it private.
 - You can declare variables, aliases, functions, and PowerShell drives for a
   scope outside of the current scope.
 - An item that you created within a scope can be changed only in the scope in
   which it was created, unless you explicitly specify a different scope.
+- When code running in a runspace references a scoped item, PowerShell searches
+  the scope hierarchy it has access to looking for a name match. If it doesn't
+  exist, a new item is created in the current scope. If it finds a match, the
+  item is copied to the current scope. If you create an item in a scope, and
```

Comment:
i _think_ it's only copied into the new scope if you alter the value (it could be _very_ resource intensive otherwise). Otherwise, we just _reference_ it in the first parent scope in which it is found. I know that's what it did in v1, but I'm not sure about that now. Patrick may know.

There may be specific instance where we _do_ copy (closures, i think), but I don't think it's the general case.